src/routes/dockerflow.js

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+// In this router we implement the services required by the dockerflow mozilla
+// best practices, as described in https://github.com/mozilla-services/Dockerflow.
+
+import Router from '@koa/router';
+import fs from 'fs';
+import path from 'path';
+
+export default function() {
```

Comment:
What do you think about exporting named functions? It seems like they are easier to refactor since you can identify them across the codebase. e.g. `export function dockerFlowRoutes () {}`

Reply:
in this PR I'm doing it for this one only but I'll follow-up for the other cases in the codebase.